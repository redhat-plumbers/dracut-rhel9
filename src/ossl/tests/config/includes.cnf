openssl_conf = openssl_init

.include = tests/config/includes.1.d
.include tests/config/includes.2.d
.include tests/config/nonexistant.d
.include tests/config/included-file.noncnf
