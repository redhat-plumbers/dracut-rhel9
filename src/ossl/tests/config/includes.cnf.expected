# This configuration file was linarized and expanded from tests/config/includes.cnf
openssl_conf = openssl_init

[included-file]
present = true

[includes1]
cnf-file = present
conf-file = present

[includes2]
main = present
